### ISSUE TITLE
Adding a Shadow

### ISSUE THREAD

**Issue by helloscoopz** [wyzantinc/vue-radial-progress#41]
Is there any way I could do a shadow underneath? something like this,
![image](https://user-images.githubusercontent.com/26799723/94040080-20093080-fde6-11ea-8292-978ffc07b3dd.png)

**Comment by mca312** [comment]
Do you have a PR for this?  Can this be done outside of the package?  I'd like to keep styling stuff out of the package as to not interfere with other users.

**Comment by helloscoopz** [comment]
No, I don't have a PR, Can't do the shadow outside because it's taking the parent div and renders like this. 
![image](https://user-images.githubusercontent.com/26799723/94047199-60b97780-fdef-11ea-9dc5-071a13859c3e.png)

anyways I've tried making the parent div rounded and that ended up like this
![image](https://user-images.githubusercontent.com/26799723/94047283-821a6380-fdef-11ea-85bf-51c18bfa4dd1.png)

any idea?
